sentry-android-core/src/main/java/io/sentry/android/core/SentryInitProvider.java

```diff
@@ -21,7 +21,9 @@ public boolean onCreate() {
       logger.log(SentryLevel.FATAL, "App. Context from ContentProvider is null");
       return false;
     }
-    if (ManifestMetadataReader.isAutoInit(context, logger)) {
+
+    if (ManifestMetadataReader.isAutoInit(context, logger)
+        && !ContextUtils.appIsLibraryForComposePreview(context)) {
```

Comment:
I'm wondering if we should do this check inside `SentryAndroid.init` actually? For the case when someone manually initializes the SDK in the library module?

Reply:
Yeah I thought about this too, my take: If you manually call init within a library module (or a dependency of that library) you very likely have a DSN set inside that lib or within code. Otherwise any `androidTest` builds would fail for that library as well. 
So I'd keep it as is for now, and if someone still has issue, we can move this block elsewhere too.